accumulo/README.md

```diff
@@ -18,6 +18,7 @@ The graph is stored in a hand full of tables with the following schema.
 <tr><td>V[id]</td>                                        <td>V</td>          <td>-</td>                                             <td>-</td>               <td>Vertex id</td></tr>
 <tr><td>V[id]</td>                                        <td>H</td>          <td>H</td>                                             <td>-</td>               <td>Vertex hidden marker</td></tr>
 <tr><td>V[id]</td>                                        <td>D</td>          <td>D</td>                                             <td>-</td>               <td>Vertex soft delete marker</td></tr>
+<tr><td>V[id]</td>                                        <td>AV</td>         <td>[visibility_string]</td>                           <td>-</td>               <td>Additional visibility to apply to the element</td></tr>
```

Comment:
The additional visibilities are just stored here using the visibility string as the column qualifier